src/test/java/uk/nhs/digital/nhsconnect/lab/results/inbound/fhir/EdifactToFhirServiceTest.java

```diff
@@ -1,5 +1,11 @@
 package uk.nhs.digital.nhsconnect.lab.results.inbound.fhir;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static uk.nhs.digital.nhsconnect.lab.results.fixtures.FhirFixtures.generateBundle;
+import static uk.nhs.digital.nhsconnect.lab.results.fixtures.FhirFixtures.generatePractitioner;
+import static uk.nhs.digital.nhsconnect.lab.results.fixtures.PathologyRecordFixtures.generatePathologyRecord;
```

Comment:
Looks like we might need to synchronise our IDE settings to stop imports bouncing around with each PR.